README.md

```diff
@@ -118,6 +118,8 @@ See the system requirements for the [OpenTok Android SDK](https://tokbox.com/dev
      @end
    </pre>
 
+   If your app uses a Swift AppDelegate file (AppDelegate.swift), you will need to have its implementation of the `RCTAppDelegate.application(_, didFinishLaunchingWithOptions)` method use a bridging header to call a method in an Objective-C++ file that calls the `[RCTComponentViewFactory registerComponentViewClass:]` method, passing in the `OTPublisherViewNativeComponentView` and `OTSubscriberViewNativeComponentView` classes.
```

Comment:
```suggestion
   If your app uses a Swift AppDelegate file (AppDelegate.swift), you will need to create an Objective-C++ class that includes a method (for example, `registerCustomComponents()`) that calls the `[RCTComponentViewFactory registerComponentViewClass:]` method twice -- to register the `OTPublisherViewNativeComponentView` and `OTSubscriberViewNativeComponentView` classes. In your Swift AppDelegate class, the implementation of the `RCTAppDelegate.application(_, didFinishLaunchingWithOptions)` method should call the method Objective-C++ class that you created.
```

Reply:
We can say for objc app do (A) , for Swift do (B). Right now both A and B are in one paragraph (sort of)

Personally I think it would be tough to follow the above suggestions and get a working registration done quickly. Folks will stumble before they get it right. A link to the work we already in samples , can be "very" helpful.

Reply:
> We can say for objc app do (A) , for Swift do (B). Right now both A and B are in one paragraph (sort of)
> 
> Personally I think it would be tough to follow the above suggestions and get a working registration done quickly. Folks will stumble before they get it right. A link to the work we already in samples , can be "very" helpful.

@IGitGotIt  I agree. I'll check with Product if they think it's OK for us to add a new-architecture test branch to the samples repo. I would want to update all of the samples in the repo, not just the BasicVideoChat and the TypeScript sample.